modules/ipcscript.js

```diff
@@ -60,8 +123,6 @@ IPCScript.prototype.info = function() {
   return {
     'uuid': this.uuid,
     'version': this.addonVersion,
```

Comment:
I did not change this code. `this.addonVersion` is [set in the constructor](https://github.com/the8472/greasemonkey/blob/9c9e5af5ceb000f093108f9f4ac6fd1529987eab/modules/ipcscript.js#L11)